flutter/example/lib/main.dart

```diff
@@ -180,6 +182,17 @@ class MainScaffold extends StatelessWidget {
               },
               child: const Text('Capture from FlutterError.onError'),
             ),
+            ElevatedButton(
+              onPressed: () {
+                (WidgetsBinding.instance.platformDispatcher as dynamic)
+                    .onError
+                    ?.call(
+                      Exception('PlatformDispatcher.onError'),
+                      StackTrace.current,
+                    );
```

Comment:
Maybe add a comment that to make this work, you need to `options.addIntegration(OnErrorIntegration());` and run on Flutter 3.3